dashboard/app/models/regional_partner_program_manager.rb

```diff
@@ -15,4 +15,6 @@
 class RegionalPartnerProgramManager < ActiveRecord::Base
   belongs_to :program_manager, class_name: 'User'
   belongs_to :regional_partner
+
+  has_many :pd_workshops, class_name: 'Pd::Workshop', through: :program_manager
```

Comment:
Organizing, via [this association](https://github.com/code-dot-org/code-dot-org/blob/a36c1ddcb536750558c640330e13395555eb1924/dashboard/app/models/user.rb#L187) on User. Perhaps, instead of `pd_workshops`, we should call it something like `pd_workshops_organizing ?

That is not set for newly created workshops. AFAIK we had a one time migration to populate it, and it's otherwise it's a placeholder for a future feature - where the UI will (eventually) give partner organizers the choice of organizing a workshop as themselves or as the partner (which could then be shared with other program managers for the same partner).